tsl/src/continuous_aggs/create.c

```diff
@@ -951,7 +951,13 @@ tsl_process_continuous_agg_viewstmt(Node *node, const char *query_string, void *
 		refresh_window.end = ts_time_get_noend_or_max(refresh_window.type);
 
 		CaggRefreshContext context = { .callctx = CAGG_REFRESH_CREATION };
-		continuous_agg_refresh_internal(cagg, &refresh_window, context, true, true, false);
+		continuous_agg_refresh_internal(cagg,
+										&refresh_window,
+										context,
+										true,
+										true,
+										false,
+										true /* move_hypertable_invalidations */);
 	}
```

Comment:
We might not always want to do this even when creating a new continuous aggregate.